src/pybind/mgr/cephadm/module.py

```diff
@@ -1389,10 +1432,12 @@ def _daemon_action(self, daemon_type, daemon_id, host, action):
         }
         name = '%s.%s' % (daemon_type, daemon_id)
         for a in actions[action]:
-            out, err, code = self._run_cephadm(
-                host, name, 'unit',
-                ['--name', name, a],
-                error_ok=True)
+            try:
+                out, err, code = self._run_cephadm(
+                    host, name, 'unit',
+                    ['--name', name, a])
+            except Exception:
+                self.log.exception('cephadm failed')
```

Comment:
being a bit more descriptive would certainly help :)

Reply:
`log.exception` will print:

- the full exceptiont traceback
- the error message of the exception

MAybe I could add the name or something.